projects/angular/src/data/datagrid/utils/key-navigation-grid.controller.ts

```diff
@@ -11,24 +11,31 @@ import { debounceTime, takeUntil } from 'rxjs/operators';
 
 import { Keys } from '../../../utils/enums/keys.enum';
 
+const actionableItems = [
```

Comment:
```suggestion
const actionableItemSelectors = [
```

I think this should be renamed to indicate that these are selectors.

Reply:
See my answer about the tabable items. I'll improve the naming.